scenes/teleporter/visible_teleporter.gd

```diff
@@ -0,0 +1,7 @@
+@tool
+extends Teleporter
+
+
+func _ready() -> void:
+	super._ready()
+	$BlueFire.play("burning")
```

Comment:
Why not add an `@onready` for this `$BlueFire` node and also add a unique name to it? We have that everywhere else.

```
@onready var blue_fire: AnimatedSprite2D = %BlueFire 
```